backend/query/test/items.e2e-spec.ts

```diff
@@ -285,9 +286,32 @@ describe('QueryController (e2e)', () => {
         await itemModel.deleteMany();
         await request(server)
           .get(queriesPath + subPath)
-          .query({ slug: relatedItems[0].slug })
+          .query({ query: relatedItems[0].tags[0].name })
```

Comment:
Why is this changed?

Reply:
because slug is not part of the DTO, since it wouldn't make sense to get statistics for a query that can remove one item at most.
This test only still worked because it can't find anything anyway.
So we could remove the entire object if we wanted but I liked it with input more for consistency